pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -317,6 +325,8 @@ buildStdenv.mkDerivation ({
      configureFlagsArray+=(
        "--enable-profile-generate=cross"
      )
+     OLD_LDFLAGS="$LDFLAGS"
+     LDFLAGS="-Wl,-rpath,$(pwd)/mozobj/dist/${binaryName}"
```

Comment:
I understand what this is doing, but not why. Could do well with a comment.

Reply:
I added comments :)